src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const populationCountry = document.querySelectorAll('span');
```

Comment:
The variable `populationCountry` is intended to select elements with the class 'population', but it currently selects all 'span' elements. This does not align with the task requirement to get text data from 'span' elements with the class 'population'.